playlists/excel.yaml

```diff
@@ -136,6 +158,19 @@
   group: Range
   api_set:
     ExcelApi: 1.1
+- id: excel-35-worksheet-reference-worksheets-by-relative-position-
```

Comment:
Are we generating such name?  with the folder name (`35-worksheet`) within it?  That seems odd...